tsconfig.json

```diff
@@ -4,6 +4,6 @@
     "outDir": "dist",
     "noEmit": true
   },
-  "include": ["packages/**/*.ts"],
+  "include": ["packages/**/*.ts", "packages/**/*.js"],
```

Comment:
Hmm, why this change? We're intentionally *not* including `.js` files here, and also don't have `checkJs: true`, so it shouldn't particularly affect the behavior of the compiler. 🤔

Reply:
@wycats Mentioned to me that this changed fixed some issues with auto-import in VSCode. If we decide to keep it, it should probably go in a separate PR.

Reply:
Yeah, totally agree. I meant to separate it out but I think we forgot.

Reply:
TL;DR if you don't include the JS files in the tsconfig then the language server doesn't do any cross-module resolving in the JS files, which:

- breaks auto-import
- causes types from other modules to not resolve, which creates a lot of inferred `any`s for no reason (and breaking intellisense)
- causes some types to resolve to the published Ember types (which are present in node_modules), which is pretty confusing.

I meant to check whether jsconfig would be good enough to resolve these issues. Since you're the TS expert here, would you prefer to add the `js` files to `tsconfig` (without `checkJS`), or try to separate them into another config file.

This could either mean `jsconfig` (if it works) or creating a second `tsconfig` and link to it from the main one with a project reference.

I'm open to anything that addresses the workflow issues, and I'm totally fine with separating it into a separate PR.

Reply:
Ah, so this specifically ends up affecting tests, which are largely unconverted? Makes sense. I'm totally fine with `js` files in the config; I just wanted to understand where they had come up since all our actual implementation is now in TS—but "tests" makes sense. We can land that right after this. Thanks for explaining!